git_machete/cmd.py

```diff
@@ -2310,11 +2373,11 @@ def prefix_dfs(u_, prefix):
         prefix_dfs(u, prefix=[])
 
     out = io.StringIO()
-    edge_color = {}
-    fp_sha_cached = {}
+    edge_color: Dict[str, str] = {}
+    fp_sha_cached: Dict[str, Optional[str]] = {}  # todo (#): default dict with None
```

Comment:
Pls open an issue + add a traditional script that would ban issue-less TODOs (can be on a separate PR ofc)

Reply:
(location: `ci/checks` + add a reference from .travis.yml)

Reply:
created #110